Springboot-demo/src/main/java/com/hollszabolcs/demo/controller/EmployeeController.java

```diff
@@ -0,0 +1,28 @@
+package com.hollszabolcs.demo.controller;
+
+import com.hollszabolcs.demo.model.Employee;
+import com.hollszabolcs.demo.service.EmployeeService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+public class EmployeeController {
+    private final EmployeeService employeeService;
+
+    @PostMapping("/employee")
+    Employee addEmployee(@RequestBody Employee employee) {
+        log.info("Adding an employee{}", employee);
+        return employeeService.addEmployee(employee);
+    }
+
+    @GetMapping("/get-employee")
```

Comment:
please get rid of this api naming violation and the empty lines in Employee.java and i will accept it

```suggestion
    @GetMapping("/employee")
```